apps/video/js/thumbnail_item.js

```diff
@@ -1,7 +1,7 @@
-/* global MediaUtils,VideoUtils */
+/* global MediaUtils,Tagged,VideoUtils */
 /**
  * ThumbnailItem is view object for a single video data. It renders video with
- * the ThumbnailItem.Template. Before use it, The template should be
+ * ThumbnailItem.view. Before use it, The template should be
```

Comment:
Nit: it seems this part of the comment no longer makes sense "Before use it, The template should be initialized."